### ISSUE TITLE
Campaign is not working with Mailchimp 

### ISSUE THREAD

**Issue by manh-nguyen** [orocrm/OroCRMMailChimpBundle#7]
Hi guys,
I have a problem with synchronize list of email campaigns between Mailchimp and OroCRM CE 1.8.

I’ve setup MailChimp integration successfully, users exported from marketing list to a segment of a subscribers list. I’ve created dome email campaigns for both(OroCRM and Mailchimp). I’ve ran syncs from UI with “Enable Two Way Sync”, daemon was running, in addition I run sync commands manually from console. But I never saw imported campaign or any statistic from MailChimp to OroCRM or from OroCRM to Mailchimp.

Mailchimp config: http://prntscr.com/8vr82e

Give me some hints please.
Thanks!
Manh - Balance Internet

**Comment by x86demon** [comment]
Hi
After Marketing List is connected to MailChimp List corresponding static segment will be created. Then  you'll be able to create and send Email Campaign in MailChimp, this campaign must be sent to static segment created by OroCRM.

Please, note that statistic is imported only for MailChimp Email Campaigns that are Sent to Static Segment created by OroCRM.

**Comment by dominicnguyen810** [comment]
Hi,

Thanks for reply.
I have imported campaigns from Mailchimp to OroCRM:

Start processing "campaign" connector
Adding new MailChimp Campaign [origin_id=d0e487097d]
Syncing Existing MailChimp Campaign [origin_id=fb99dc0adc]
Adding new MailChimp Campaign [origin_id=b382368447]
Stats: read [3], processed [2], updated [0], added [2], deleted [0], invalid entities: [0]

But  it does not show in OroCRM(Marketing->Email Campaigns). I tried to check in table orocrm_mailchimp_campaign saw MailChimp campaigns are saved here.

So, how to manager Mailchimp Campaigns in OroCRM?

**Comment by x86demon** [comment]
Email Campaign in Oro is created by process. Is Job Daemon started and all jobs are finished?

**Comment by dominicnguyen810** [comment]
Yes, I have ran it in Job Daemon and run command manually from console. All finished without error.

**Comment by dominicnguyen810** [comment]
Hi guys,

Do you have any ideas?

**Comment by rogyar** [comment]
Unfortunately, have absolute the same problem 
```
Campaign Connector Stats: read [0], processed [0], updated [0], added [0], deleted [0], invalid entities: [0]
```

But we have 10+ campaigns in MainChimp

**Comment by WillRadi** [comment]
Same problem... 
I'm using OroCRM 2.6

**Comment by WillRadi** [comment]
Hi @mbessolov !
In background, is running `oro:message-queue:consume` and `clank:server`
Is there another one?

There is no errors in log files. Just info.

**Comment by arturas-bes** [comment]
> Hi @mbessolov !
> In background, is running `oro:message-queue:consume` and `clank:server`
> Is there another one?
> 
> There is no errors in log files. Just info.

Have you figuret out the problem on this mailchimp?

**Comment by mbessolov** [comment]
@nachaliuga Mailchimp works fine in 2.6
If you experience the same problem - please detailed steps on how to reproduce it on a clean installation, test data set, errors from the application logs, etc.
